examples/platform/esp32/ota/OTAHelper.cpp

```diff
@@ -74,8 +74,13 @@ static void InitOTARequestorHandler(System::Layer * systemLayer, void * appState
 
 void OTAHelpers::InitOTARequestor()
 {
-    chip::DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Seconds32(kInitOTARequestorDelaySec), InitOTARequestorHandler,
-                                                nullptr);
+    static bool isOTAInitialized = false;
+    if (!isOTAInitialized)
+    {
+        chip::DeviceLayer::SystemLayer().StartTimer(chip::System::Clock::Seconds32(kInitOTARequestorDelaySec),
```

Comment:
Since we are now waiting for `kDnssdInitialized`, do we still need the additional 3-second delay here?  Maybe consider taking it out in a followup, after some testing.

Reply:
Done